File: library/src/main/java/com/github/droibit/messenger/MessageCallback.java
package com.github.droibit.messenger;

import android.support.annotation.NonNull;

import com.google.android.gms.common.api.Status;

/**
 * Callback of send message using the Message API.
 */
public interface MessageCallback {

    /**
     * Called in the message send.
     *
     * @param status send result
     */
    void onMessageResult(@NonNull Status status);
}

File: library/src/main/java/com/github/droibit/messenger/MessageCallback.kt
package com.github.droibit.messenger

import com.google.android.gms.common.api.Status

/**
 * Callback of send message using the Message API.
 */
interface MessageCallback {

    /**
     * Called in the message send.
     *
     * @param status send result
     */
    fun onMessageResult(status: Status)
}
